packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_details/paths.dart

```diff
@@ -126,8 +125,33 @@ class RetainingPathTable extends StatelessWidget {
   final bool isDiff;
   final ClassData classData;
 
-  late final _data =
-      classData.byPath.keys.map((path) => PathData(classData, path)).toList();
+  @visibleForTesting
+  static void resetSingletons() {
```

Comment:
On second thought, I don't think we should have this timing code here. If we want to have a test that test covers the time that `classData.byPath.keys.map((path) => PathData(classData, path)).toList();` takes to run, we should call that directly in the test case and we can remove all this testing-specific code here.